examples/shopify/README.md

```diff
@@ -0,0 +1,201 @@
+# Demo
+
+[https://makeswift-examples-shopify.vercel.app](https://makeswift-examples-shopify.vercel.app)
+
+# Visually build with Shopify components
+
+In this example, you will learn how to integrate [Shopify](https://www.shopify.com/) with [Makeswift](https://www.makeswift.com) to create a visually editable ecommerce store.
+
+This example includes a home page for listing products by category and a product template page for including all the details.
+
+## Tools
+
+- [**Shopify StoreFrontAPI**](https://shopify.dev/api/storefront): to pull data related to a store in Shopify and provide it via React context.
+- [**Makeswift SDK**](https://www.makeswift.com/docs): to register components into Makeswift's visual builder.
+
+---
+
+## Using this repo
+
+1. **Create a site in Makeswift and a shop in Shopify**
+
+   Head over to [Makeswift](https://app.makeswift.com) and sign up for a free account. Create a site using the option to "Integrate with Next.js",
+   and copy the Site API key. You will need it in step 3.
+
+   Then head over to [Shopify](https://www.shopify.com/) and create a store. Here is [a good starting point](https://www.shopify.com/online) to get you started.
+
+2. **Clone this template**
+
+   Instead of using `create-next-app`, run this command from the terminal:
+
+   ```bash
+   npx degit makeswift/makeswift/examples/shopify  shopify
+   cd shopify
+   ```
+
+   It will download this subdirectory of the `makeswift/makeswift` repo without including git history
+
+3. **Update environment variables**
+
+   Rename `.env.local.example` to `.env.local` and update it with values from your Shopify and Makeswift accounts.
+
+   - `SHOPIFY_STORE_NAME` can be found in the Shopify dashboard
+
+       <img src="https://user-images.githubusercontent.com/20950876/184916524-667084c1-06a0-4fa3-8f4b-73aff5a88e65.png" width="600" />
+
+   - `SHOPIFY_ACCESS_TOKEN` requires you to [register an app](https://www.shopify.com/partners/blog/17056443-how-to-generate-a-shopify-api-token)
+   - and finally the `MAKESWIFT_SITE_API_KEY` comes from your Makeswift site. It is the value you copied from step 1.
+
+   ```diff
+   - SHOPIFY_STORE_NAME=
+   - SHOPIFY_ACCESS_TOKEN=
+
+   - MAKESWIFT_SITE_API_KEY=
+   + SHOPIFY_STORE_NAME=<YOUR_SHOPIFY_STORE_NAME>
+   + SHOPIFY_ACCESS_TOKEN=<YOUR_SHOPIFY_ACCESS_TOKEN>
+
+   + MAKESWIFT_SITE_API_KEY=<YOUR_MAKESWIFT_SITE_API_KEY>
+   ```
+
+4. **Start the dev server**
+
+   Run this command from the terminal:
+
+   ```bash
```

Comment:
"this command" isn't copy/paste-able or runnable in the terminal. We should probably just use a single type of package manager here. Which one should be based on what package manager the example is using. All our examples seem to have a Yarn lockfile so we should use Yarn